### ISSUE TITLE
no vcf-annotate commander with conda installation

### ISSUE THREAD

**Issue by xiucz** [vcftools/vcftools#96]
I installed vcftools using conda(conda version 4.3.23), it succeed.  And the _**vcftools**_ commander works well. But I find no vcf-annotate and other commanders here.

```
ls ~/software/anaconda3/pkgs/vcftools-0.1.15-0/bin/

vcftools
```
Are there any explainations? Thank you.

**Comment by florealcab** [comment]
Same problem for me... Do you have an explanation or a fix?

**Comment by florealcab** [comment]
It works well if I ask version 0.1.11.*, but not with the latest (0.1.15)

**Comment by yjarosz** [comment]
Hello,
I have the same issue trying `vcf-convert`. It seems that https://github.com/bioconda/bioconda-recipes/pull/4773 break the installation of the perl scripts.

I think the line `sed -i.bak -e 's/SUBDIRS = cpp perl/SUBDIRS = cpp/' ./src/Makefile` in the current bioconda installation does not install the perl modules anymore. The perl module is now available via the `perl-vcftools-vcf` pakage but this one also doesn't install the perl scripts.

**Comment by yjarosz** [comment]
Maybe the `perl-vcftools-vcf` should also install the scripts by running the Makefile?
Something like this could be added to the https://github.com/bioconda/bioconda-recipes/blob/master/recipes/perl-vcftools-vcf/build.sh maybe ?

```
#!/bin/bash
set -euo pipefail
export CPPFLAGS="-I$PREFIX/include"
export LDFLAGS="-L$PREFIX/lib"
./autogen.sh
./configure --prefix=$PREFIX
sed -i.bak -e 's/SUBDIRS = cpp perl/SUBDIRS = perl/' ./src/Makefile
make
make install
```

I'm not sure it can help 😝

**Comment by xiucz** [comment]
@yjarosz Thank-you!

**Comment by bounlu** [comment]
I have the same problem as of today. How to fix?

**Comment by xiucz** [comment]
@bounlu 

Sorry for the late reply, did you successed ?  I followed yjarosz's advice and fixed it. Good luck

**Comment by bounlu** [comment]
@xiucz 
I tried to add it to the `build.sh` file and re-installed `perl-vcftools-vcf` but it didn't help. Could you please elaborate the steps how you managed to fix it? A copy paste of the `build.sh` file would be very useful as well.

**Comment by pd3** [comment]
Please switch from `vcf-annotate`, which is obsolete, to `bcftools annotate`, which is much faster, has more features and is supported. (I am the author of both.) 
http://samtools.github.io/bcftools/
http://samtools.github.io/bcftools/bcftools.html#annotate

**Comment by bounlu** [comment]
@pd3 
Thanks Peter, I already have bcftools installed, and I was actually after `vcf-subset`, and the equivalent tool in vcftools is `bcftools view`.

**Comment by deeptimittal12** [comment]
If I use conda install perl-vcftools-vcf command, it installs the latest version 0.953 but all the perl scripts are missing. I tried to specify the version in the command: conda install -c bioconda perl-vcftools-vcf=0.1.16 but vcf-convert and some other perl scripts are still missing. I want to use vcf-convert. Does anyone here knows about the equivalent of vcf-convert in bcftools? I checked but did not find one, Thanks a lot.
